crypto/src/merkle/proofs.rs

```diff
@@ -255,6 +255,158 @@ impl<H: Hasher> BatchMerkleProof<H> {
         v.remove(&1).ok_or(MerkleTreeError::InvalidProof)
     }
 
+    /// Computes the uncompressed Merkle paths which aggregate to this proof.
+    ///
+    /// # Errors
+    /// Returns an error if:
+    /// * No indexes were provided (i.e., `indexes` is an empty slice).
+    /// * Number of provided indexes is greater than 255.
```

Comment:
We should probably add another error condition here which is when the number of indexes does not match the number of leaf nodes in the proof.